src/editor-page.jsx

```diff
@@ -88,12 +125,28 @@ var EditorPage = React.createClass({
                     ref="hintsEditor"
                     hints={this.props.hints}
                     imageUploader={this.props.imageUploader}
-                    onChange={this.handleChange} />
+                    onChange={this.handleChange}
+                    searchString={this.state.searchString}
+                    searchIndex={combinedHintsSearchIndex} />
+            }
+
+            {(this.props.searchAndReplace) &&
+                <SearchAndReplaceDialog
+                    ref="searchAndReplace"
+                    question={this.props.question}
+                    answerArea={this.props.answerArea}
+                    hints={this.props.hints}
+                    onChange={this.handleStateChange}
```

Comment:
There's a ```handleChange``` method but it does some stuff with props but I needed to modify the state.